app/src/main/java/io/spaceapi/community/myhackerspace/Main.java

```diff
@@ -625,6 +636,18 @@ private void addSensor(
         } else {
             viewDetails2.setVisibility(View.GONE);
         }
+        if (lastchange != null) {
+            CharSequence relativeLc = DateUtils.getRelativeDateTimeString(
+                this,
+                (long) (lastchange * 1000),
+                DateUtils.MINUTE_IN_MILLIS,
+                DateUtils.WEEK_IN_MILLIS,
+                DateUtils.FORMAT_ABBREV_RELATIVE
+            );
```

Comment:
This seems to have issues with timestamps that are slightly in the future:

![image](https://github.com/user-attachments/assets/3012e632-61fe-487e-bb23-d140c623150c)

Initially it even showed "in 1 minute".

Suggestion: If `timestamp > (now - 1min)`, show "Moments ago" as last update.

Reply:
Fixed/implemented in 55b0497

Reply:
If people put there `last_change` into the future I'd actually expect the app to show it as in the future.

Reply:
Suggestion a compromise: use e.g. "now" for `abs(now - lastchange) < 60s`, and use the relative timestamp otherwise.

Reply:
Implemented